content/es/docs/reference/glossary/replica-set.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ReplicaSet
+id: replica-set
+date: 2018-05-16
+full_link: /docs/concepts/workloads/controllers/replicaset/
+short_description: >
+  El ReplicaSet es la nueva generación del ReplicationController.
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ El ReplicaSet es la nueva generación del ReplicationController.
+
+<!--more--> 
+
+Un ReplicaSet, análogamente a un ReplicationController, garantiza que un número establecido de replicas de un pod estan corriendo en un momento dado. El ReplicaSet soporta definir los nuevos requisitos de selector del tipo set-based (basados en grupo), mientras que el ReplicationController tan sólo soporta selectores equality-based (basados en igualdad).
```

Comment:
```suggestion
Un ReplicaSet, análogamente a un {{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}}, garantiza que un número establecido de réplicas de un pod están corriendo en un momento dado. El ReplicaSet tiene soporte para selectores del tipo set-based, lo que permite el filtrado de claves por grupos de valores como por ejemplo todos los pods cuya etiqueta no `environment` no sea `production` ni `qa`. Por otro lado, el ReplicationController sólo soporta selectores equality-based, es decir, que solo puedes filtrar por valores exactos como por ejemplo, los pods que tengan la etiqueta `tier` con valor `frontend`-
```

El tema de los selectores creo que cuesta de entender si no conoces los conceptos, no sé si tiene sentido explicar los conceptos en esta definición, pero ayuda a entender la diferencia. Up to you.

Reply:
Con los ejemplos me parece más facil de entender, sin duda :)